deploy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/operator-framework/upstream-registry-builder as builder
```

Comment:
Be sure to create a `.dockerignore` file and add `deploy/` to it (so the Operator container SHA is unaffected by changes to this directory).